src/ontology/uberon-edit.obo

```diff
@@ -224736,18 +224751,53 @@ relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
 property_value: dcterms-date 2023-02-10T14:54:06Z xsd:dateTime
 
 [Term]
-id: UBERON:8480047
-name: retroauricular region
-def: "The retroauricular region is the area located behind the auricle of the outer ear." [http://orcid.org/0000-0002-0819-0473]
-synonym: "retro-auricular area" EXACT [PMID:22592943]
-synonym: "retro-auricular part of head" EXACT [FMA:322167]
-synonym: "retro-auricular region" EXACT [PMID:33988139]
-synonym: "retroauricular area" EXACT [PMID:29470606]
-is_a: UBERON:0000061 ! anatomical structure
-relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
-relationship: part_of UBERON:0000033 ! head
-property_value: curator_notes "Exists at least in all mammals. https://www.imaios.com/en/vet-Anatomy/Vet-Anatomical-Part/Retroauricular-region" xsd:string
-property_value: dcterms-date 2023-02-09T09:39:18Z xsd:dateTime
+id: UBERON:8600004
+name: visceral muscle tissue
+def: "A muscle tissue that is associated with soft internal organs (viscera)." [BTO:0002780, CLAO:0001295]
+comment: It is quite common to refer to visceral muscles as smooth muscles, by analogy with vertebrates; however, most visceral muscles in arthropods are actually striated. {xref="https://en.wikipedia.org/wiki/Smooth_muscle", xref="FBbt:00005070"}
```

Comment:
PMID:8720463 seems to support this comment, perhaps more than the current dbxrefs:

"Of all invertebrate muscles, the transversely striated muscle with continuous Z lines is the most similar to the vertebrate skeletal muscle and is present in arthropods, whose musculature (including the visceral muscles) only consists of this cell type. "